### PR TITLE
Feature LLE 262 - Add NPS Campaign popup style

### PR DESCRIPTION
**Pull request by DimitrijeMihic**

### CODE REVIEW

**Review by nothingrandom — COMMENTED** [review]
Why is everything `!important`? What is it we're having to override this way?

**Comment by DimitrijeMihic** [comment]
We're injecting elements directly in the page, and the page has its own style, different from ours. In most cases the page style overrides our style, that's why we have `!important` in almost every line.

File: src/workflow.scss [hunk 1]
```diff
@@ -833,6 +835,99 @@
...
+  overflow-y: auto !important;
+  margin: auto;
+}
+
+.workflow-extension-campaign-question {
+  margin-bottom: 20px !important;
+  padding: 15px !important;
+  border-radius: 20px !important;
+  background-color: #0093D5;
+  color:#fff !important;
```
Comment (nothingrandom) [inline src/workflow.scss]:
```suggestion
  color: #fff !important;
```

File: src/workflow.scss [hunk 2]
```diff
@@ -833,6 +835,99 @@
...
+.workflow-extension-campaign-question-feedback {
+  display: flex !important;
+  justify-content: space-evenly !important;
+  background: #fff !important;
+  border-radius: 20px !important;
+  padding: 8px !important;
+}
+
+.workflow-extension-campaign-question-feedback input[type="radio"] {
+  display:none !important;
```
Comment (nothingrandom) [inline src/workflow.scss]:
```suggestion
  display: none !important;
```

File: src/workflow.scss [hunk 3]
```diff
@@ -891,53 +986,58 @@
...
-    align-items: center !important;
+  width:100%;
+  color:#3B3B3B;
+  text-decoration: none !important;
+  font-weight: 500 !important;
+  font-family: 'Montserrat' !important;
+  font-size:14px !important;
+  display:flex !important;
+  align-items: center !important;
+  border: none !important;
```
Comment (nothingrandom) [inline src/workflow.scss]:
```suggestion
  width: 100%;
  color: #3B3B3B;
  text-decoration: none !important;
  font-weight: 500 !important;
  font-family: 'Montserrat' !important;
  font-size: 14px !important;
  display: flex !important;
  align-items: center !important;
  border: none !important;
```

File: src/workflow.scss [hunk 4]
```diff
@@ -891,53 +986,58 @@
...
-    font-weight: 500 !important;
-    font-family: 'Montserrat' !important;
-    font-size:14px !important;
+  width:100%;
+  color:#3B3B3B;
+  text-decoration: none !important;
+  font-weight: 500 !important;
+  font-family: 'Montserrat' !important;
+  font-size:14px !important;
+  border: none !important;
```
Comment (nothingrandom) [inline src/workflow.scss]:
```suggestion
  width: 100%;
  color: #3B3B3B;
  text-decoration: none !important;
  font-weight: 500 !important;
  font-family: 'Montserrat' !important;
  font-size: 14px !important;
  border: none !important;
```

File: src/workflow.scss [hunk 5]
```diff
@@ -891,53 +986,58 @@
...
+  color: rgb(96, 194, 240) !important;
+  font-weight: bold !important;
+  display: inline-block !important;
+  width: 0.8em !important;
+  vertical-align: top !important;
 }
 .workflow-extension-quickmenu-right-content-item ul li a{
-    font-size:13px !important;
-    transition: .3s !important;
+  font-size:13px !important;
```
Comment (nothingrandom) [inline src/workflow.scss]:
```suggestion
  font-size: 13px !important;
```

**Review by nothingrandom — CHANGES_REQUESTED** [review]
Mentions of `font-family: 'Montserrat';` would make more sense as;

```scss
$font-sans: 'Montserrat', sans-serif;
font-family: $font-sans;
```

**Review by nothingrandom — APPROVED** [review]

**Review by DraganaBelic — APPROVED** [review]

**Pull request merged**